packages/core/admin-client/src/authProviders/ProviderConfigs.ts

```diff
@@ -49,7 +49,13 @@ class UserpassProviderConfig {
     
     @jsonProperty("confirmEmailSubject") public confirmEmailSubject: string,
     
-    @jsonProperty("resetPasswordSubject") public resetPasswordSubject: string
+    @jsonProperty("resetPasswordSubject") public resetPasswordSubject: string,
+
+    @jsonProperty("runResetFunction") public runResetFunction = false,
+
+    @jsonProperty("resetFunctionId") public resetFunctionId = "",
+
+    @jsonProperty("resetFunctionName") public resetFunctionName = ""
```

Comment:
while you do this, you should probably add the fields for confirmation function as well 
```
"runConfirmationFunction"
"confirmationFunctionId"
"confirmationFunctionName"
```


https://github.com/10gen/stitch/blob/a0a456ed1189ea3a6fecdf6e40f222a1f43618df/authprovider/providers/local/userpass.go#L76